Formula/tinysearch.rb

```diff
@@ -0,0 +1,22 @@
+class Tinysearch < Formula
+  desc "Tiny, full-text search engine for static websites built with Rust and Wasm"
+  homepage "https://github.com/tinysearch/tinysearch"
+  url "https://github.com/tinysearch/tinysearch/archive/refs/tags/v0.6.0.tar.gz"
+  sha256 "a166ea3f0a5f0bfd490cb36b3360b51e40a278e138f40a4be601faf4d33f456b"
+  license any_of: ["Apache-2.0", "MIT"]
+  head "https://github.com/tinysearch/tinysearch.git", branch: "master"
+
+  depends_on "rust" => :build
+  depends_on "wasm-pack"
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: "bin")
+    pkgshare.install "fixtures"
+  end
+
+  test do
+    output = shell_output("tinysearch #{pkgshare}/fixtures/index.json 2>&1", 1)
+    assert_match "Compiling WASM module using wasm-pack", output
+    assert_match "failed to execute \"wasm-pack\" \"build\"", output
```

Comment:
Is this really expected? It doesn't seem like a good sign.

Reply:
I was thinking to add `wasm-pack` as test dependency, but it did not quite help with compilation, but I can use the tinysearch in my local just fine though.

Reply:
You already have a `wasm-pack` dependency though. Is this really what you get from running the test case outside of `brew`?

Reply:
Can you try setting up `rustup-init` like in `wasm-pack` formula?
```
    system "#{Formula["rustup-init"].bin}/rustup-init", "-y", "--no-modify-path"
    ENV.prepend_path "PATH", HOMEBREW_CACHE/"cargo_cache/bin"
```

Issue is that Rust/cargo isn't set up by the time testcase is running.

Reply:
I will give another spin today.